.gitignore

```diff
@@ -3,4 +3,5 @@ apps/*/dist/
 node_modules/
 *.pem
 .env
-diff_tmp.png
\ No newline at end of file
+diff_tmp.png
+yarn.lock
```

Comment:
We do not use yarn any more in the project, so it is off topic in the `.gitignore`.

Reply:
Well, sometimes I forget about that. It was a convenient attempt to prevent regression